src/vite-env.d.ts

```
@@ -78,6 +87,13 @@ declare global {
         setOptions(options: Partial<PolylineOptions>): void;
       }
       
+      class LatLngBounds {
+        constructor();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'LatLngBounds'. Consider to delete it or rename it.

'LatLngBounds' is defined here:


(lint/suspicious/noRedeclare)